src/marks/density.js

```diff
@@ -129,7 +128,7 @@ function densityInitializer(options, fillDensity, strokeDensity) {
     // If explicit thresholds were not specified, find the maximum density of
     // all grids and use this to compute thresholds.
     let T = thresholds;
-    if (!isTypedArray(T)) {
+    if (!(T instanceof TypedArray)) {
```

Comment:
(Taking your advice on using instanceof directly… though I think the performance will be the same and maybe it’s slightly more readable to have the helper. I’m happy either way.)